modules/manage/pages/security/fips-compliance.adoc

```diff
@@ -0,0 +1,42 @@
+= Configure Redpanda for FIPS 
+:description: Configure Redpanda to operate in FIPS compliance mode.
+:page-aliases: security:fips-compliance.adoc
+:page-categories: Management, Security
+
+Redpanda platform (Redpanda Enterprise) provides FIPS-compliant cipher enforcement for brokers using OpenSSL 3.0.9, which is https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4282[validated^] for https://csrc.nist.gov/pubs/fips/140-2/upd2/final[140-2^] and is undergoing validation by NIST for https://csrc.nist.gov/pubs/fips/140-3/final[140-3^]. Both Redpanda and `rpk` leverage validated Open SSL libraries for all security-related cryptography operations.
+
+== Prerequisites
+
+Before configuring brokers to run in FIPS compliance mode (FIPS mode), check to make sure the `redpanda-rpk-fips` and `redpanda-fips` packages are xref:deploy:deployment-option/self-hosted/manual/production/dev-deployment.adoc#install-fips[installed]. These packages are required by both the `redpanda` and `redpanda-tuners` install packages.
+
+== Configure FIPS
+
+When you configure a broker to run in FIPS mode:
+
+* FIPS compliance is enforced _immediately_ across all services upon startup.
```

Comment:
What do you mean by "services"?  It is correct that FIPS compliance is enforced immediately at start of Redpanda application start up

Reply:
@michael-redpanda wouldn't services refer to Kafka API, Schema Registry API, and Proxy API? Technically they are all the same process but historically are different services.

Reply:
@deniscoady Yes that's what I thought it meant :). Just asking for clarification.